Packs/ServiceNow/Integrations/ServiceNowv2/README.md

```diff
@@ -138,15 +138,13 @@ These permissions may not suffice for managing records in some tables. Make sure
 have the correct role so you have permissions to work with the relevant table.  
 
 ### Configure Incident Mirroring When the Trigger Incident is ServiceNow  
-When the trigger incident is ServiceNow, you use the default **ServiceNow Create Ticket and Mirror**
-incident type. You can also use the default incoming and outgoing mappers or optionally create custom
-mappers.
+When the trigger incident is ServiceNow, you use the **ServiceNow Classifier** and leave the Incident type as N/A, with either the default incoming and outgoing mappers or optional custom mappers.
```

Comment:
Why did we change it to classifier and not leave the incident type?

Reply:
That was @synth-humbert 's recommendation - also, not to have both classifier and incident types selected -  _Its an either or, not both. Recommendation to use classifier not incident type as it gives more control._